ivy/functional/backends/paddle/elementwise.py

```diff
@@ -517,6 +361,18 @@ def fmin(
     return paddle.fmin(x1, x2)
 
 
+@with_supported_dtypes(
+    {
+        "2.5.1 and below": (
+            "bool",
+            "float",
+            "int32",
+            "int64",
+            "complex",
```

Comment:
have you tested if `int16` is supported for the function? it doesn't seem to be getting manually type casted in the original implementation